thundra/wrappers/django/django_executor.py

```diff
@@ -0,0 +1,46 @@
+import uuid
+
+from thundra import constants
+from thundra.wrappers import wrapper_utils, web_wrapper_utils
+
+
+def start_trace(plugin_context, execution_context, tracer):
+    request = execution_context.platform_data['request']
+
+    request = {
+        'method': request.method,
+        'host': request.get_host().split(':')[0],
+        'query_params': request.GET,
+        'body': request.body,
+        'headers': request.headers,
+        'path': request.path
+    }
+    web_wrapper_utils.start_trace(execution_context, tracer, 'Django', 'API', request)
+
+
+def finish_trace(execution_context):
```

Comment:
Can we create a `finish_trace` method in `web_wrapper_utils` and move this logic to there? So `finish_trace` here delegates to the `web_wrapper_utils.finish_trace`